lib/peg.d.ts

```diff
@@ -1197,6 +1211,30 @@ export function generate(
   options: SourceBuildOptions<"source">
 ): string;
 
+/**
+ * Returns the generated source code as a string appended with a source map as
+ * a `data:` URI.
+ *
+ * Note, that `SourceNode.source`s of the generated source map will depend
+ * on the `options.grammarSource` value. Therefore, value `options.grammarSource`
+ * will propagate to the `sources` array of the source map. That array MUST
+ * contain a path relative to the source map location, as no further path
+ * processing will be performed.
```

Comment:
That part seems incorrect. What location the inline source map have?

Reply:
This needs to be worded better.  The important point is that the grammarSource is a string with a path relative to where the .js file will eventually be written.